andela/wg-silicon
# [Feature #158626105] Upgrade the nutrition diary feature

### What does this PR do?
Allows user to add either the food eaten or the food the user plans to eat

### Description of Task to be completed?
User is able to add either the food eaten or the food the user plans to eat

### How should this be manually tested?
1. Visit `https://wg-silicon.herokuapp.com` and login
2. Go to `Nutrition plans` under `Nutrition` dropdown.
3. Click on the green `Add new nutrition plan` button
4. Click on `Add a new meal` link
5. Notice that a user can select either 'Eaten' or 'Planned' in Meal choice

### What are the relevant pivotal tracker stories?
- [Upgrade the nutrition diary feature](https://www.pivotaltracker.com/story/show/158626105)

### Screenshots
- __Before Feature was added__
<img width="660" alt="screen shot 2018-07-24 at 09 45 20" src="https://user-images.githubusercontent.com/29580769/43121797-a676a9b6-8f27-11e8-855e-74812ade7111.png">

- __With the feature added__
<img width="661" alt="screen shot 2018-07-24 at 09 46 05" src="https://user-images.githubusercontent.com/29580769/43121811-b141675a-8f27-11e8-99b2-7847c1bb485a.png">
